src/main/java/uk/ac/sanger/sccp/stan/repo/LabwareNoteRepo.java

```diff
@@ -13,4 +14,5 @@ public interface LabwareNoteRepo extends CrudRepository<LabwareNote, Integer> {
     List<LabwareNote> findAllByOperationIdIn(Collection<Integer> opIds);
     List<LabwareNote> findAllByPlanIdIn(Collection<Integer> planIds);
     List<LabwareNote> findAllByLabwareIdInAndName(Collection<Integer> labwareIds, String name);
+    boolean existsByNameAndValue(String name, String value);
```

Comment:
What is this function for?

Reply:
It is for looking up if a given run name has already been used. If it has, then `existsByNameAndValue("run", givenRunName)` will be true.

Reply:
Ok 👍